### PR TITLE
Added Bosnian translations for tour and cheatsheet

### PR DESCRIPTION
**Pull request by sake92**
Added Bosnian translations for following:
- tutorials/tour
- cheatsheet

Since Croatian and Serbian are very similar languages to Bosnian, they will also benefit from this, I hope. :)

### CODE REVIEW

**Comment by SethTisue** [comment]
called for reviewers on Gitter (https://gitter.im/scala/contributors?at=57c5d06eee799d3329f364f0)

**Comment by sjrd** [comment]
Ping @vjovanov. Maybe you can help, here?

**Comment by vjovanov** [comment]
I can review this PR thoroughly in two days. From the first look it looks very well written (grammar, integration of english terms, completeness).

File: ba/cheatsheets/index.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+|                                                                                                          |                 |
+| ------                                                                                                   | ------          |
+|  <span id="variables" class="h2">varijable</span>                                                                       |                 |
+|  `var x = 5`                                                                                             |  varijabla       |
+|  <span class="label success">Dobro</span> `val x = 5`<br> <span class="label important">Loše</span> `x=6`  |  konstanta       |
+|  `var x: Double = 5`                                                                                     |  eksplicitni tip  |
+|  <span id="functions" class="h2">funkcije</span>                                                                       |                 |
+|  <span class="label success">Dobro</span> `def f(x: Int) = { x*x }`<br> <span class="label important">Loše</span> `def f(x: Int)   { x*x }` |  definicija funkcije <br> skrivena greška: bez = ovo je procedura koja vraća Unit; uzrokuje zabunu |
```
Comment (vjovanov) [inline ba/cheatsheets/index.md]: bez = ~> bez `=`

File: ba/cheatsheets/index.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+|  `(x:R) => x*x`                                                                                          |  anonimna funkcija  |
+|  `(1 to 5).map(_*2)` vs.<br> `(1 to 5).reduceLeft( _+_ )`                                                |  anonimna funkcija: donja crta odgovara argumentu po poziciji. |
+|  `(1 to 5).map( x => x*x )`                                                                              |  anonimna funkcija: da bi koristili argument više od jednom, morate mu dati ime. |
+|  <span class="label success">Dobro</span> `(1 to 5).map(2*)`<br> <span class="label important">Loše</span> `(1 to 5).map(*2)` |  anonimna funkcija: vezana infiksna metoda. Koristite `2*_` zbog jasnoće. |
+|  `(1 to 5).map { x => val y=x*2; println(y); y }`                                                             |  anonimna funkcija: blokovski stil vraća vrijednost zadnjeg izraza. |
+|  `(1 to 5) filter {_%2 == 0} map {_*2}`                                                                  |  anonimne funkcije: pipeline stil. (može i sa oblim zagradama). |
```
Comment (vjovanov) [inline ba/cheatsheets/index.md]: Dot (`.`) goes into parenthesis. M is uppercase at the beginning of that sentence.

File: ba/cheatsheets/index.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+|  `def compose(g:R=>R, h:R=>R) = (x:R) => g(h(x))` <br> `val f = compose({_*2}, {_-1})`                   |  anonimne funkcije: da bi proslijedili više blokova, potrebne su dodatne zagrade. |
+|  `val zscore = (mean:R, sd:R) => (x:R) => (x-mean)/sd`                                                   |  curry-jevanje, očita sintaksa. |
+|  `def zscore(mean:R, sd:R) = (x:R) => (x-mean)/sd`                                                       |  curry-jevanje, očita sintaksa. |
+|  `def zscore(mean:R, sd:R)(x:R) = (x-mean)/sd`                                                           |  curry-jevanje, sintaksni šećer(kratica). ali onda: |
+|  `val normer = zscore(7, 0.4) _`                                                                          |  je potrebna prateća donja crta za parcijalnu primjenu, samo kod šećer(skraćene) verzije. |
```
Comment (vjovanov) [inline ba/cheatsheets/index.md]: šećer(skraćene) ~> šećer (skraćene)

File: ba/cheatsheets/index.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+|  <span id="packages" class="h2">paketi</span>                                                                         |                 |
+|  `import scala.collection._`                                                                             |  džoker(wildcard) import. |
+|  `import scala.collection.Vector` <br> `import scala.collection.{Vector, Sequence}`                      |  selektivni import. |
+|  `import scala.collection.{Vector => Vec28}`                                                             |  preimenujući import. |
+|  `import java.util.{Date => _, _}`                                                                       |  import svega iz java.util paketa osim Date. |
+|  `package pkg` _na početku fajla_ <br> `package pkg { ... }`                                             |  deklaracija paketa. |
+|  <span id="data_structures" class="h2">strukture podataka</span>                                                           |                 |
+|  `(1,2,3)`                                                                                               |  n-torka(tuple) literal. (`Tuple3`) |
```
Comment (vjovanov) [inline ba/cheatsheets/index.md]: torka (tuple)

File: ba/cheatsheets/index.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+|  `var (x,y,z) = (1,2,3)`                                                                                 |  destrukturirajuće vezivanje: otpakivanje n-torke podudaranjem uzorka(pattern matching). |
```
Comment (vjovanov) [inline ba/cheatsheets/index.md]: In Serbian it would be `destruktivno`, not sure about Bosnian.

File: ba/cheatsheets/index.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+|  `if (check) happy` _isto kao_ <br> `if (check) happy else ()`                                            |  sintaksni šećer za uslov. |
+|  `while (x < 5) { println(x); x += 1}`                                                                   |  while petlja. |
+|  `do { println(x); x += 1} while (x < 5)`                                                                |  do while petlja. |
+|  `import scala.util.control.Breaks._`<br>`breakable {`<br>`    for (x <- xs) {`<br>`        if (Math.random < 0.1) break`<br>`    }`<br>`}`|  break. ([slajdovi](http://www.slideshare.net/Odersky/fosdem-2009-1013261/21)) |
+|  `for (x <- xs if x%2 == 0) yield x*10` _isto kao_ <br>`xs.filter(_%2 == 0).map(_*10)`                    |  for comprehension: filter/map |
+|  `for ((x,y) <- xs zip ys) yield x*y` _isto kao_ <br>`(xs zip ys) map { case (x,y) => x*y }`              |  for comprehension: destrukturirajuće vezivanje |
+|  `for (x <- xs; y <- ys) yield x*y` _isto kao_ <br>`xs flatMap {x => ys map {y => x*y}}`                  |  for comprehension: međuproizvod(vektorski proizvod) |
```
Comment (vjovanov) [inline ba/cheatsheets/index.md]: d (vektorski

File: ba/tutorials/tour/abstract-types.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+disqus: true
+
+tutorial: scala-tour
+num: 22
+outof: 33
+language: ba
+---
+
+U Scali, klase su parameterizovane vrijednostima(parameteri konstruktora) i tipovima(ako su [generičke](generic-classes.html)).
```
Comment (vjovanov) [inline ba/tutorials/tour/abstract-types.md]: I will not comment on space before parenthesis any more. It should be fixed with search and replace.

File: ba/tutorials/tour/abstract-types.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+ 
+    trait Buffer {
+      type T
+      val element: T
+    }
+ 
+*Apstraktni tipovi* su tipovi čiji identitet nije precizno definisan.
+U gornjem primjeru, poznato je samo da svaki objekat klase `Buffer` ima a tip-član `T`, 
+ali definicija klase `Buffer` ne kazuje kojem konkretno tipu odgovara `T`. 
+Kao i definicije vrijednosti, možemo prebrisati(override) definicije tipova u podklasama.
```
Comment (vjovanov) [inline ba/tutorials/tour/abstract-types.md]: override is not `prebrisati`.

File: ba/tutorials/tour/abstract-types.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+             type T = List[U]
+             val element = List(elem1, elem2)
+           }
+      val buf = newIntSeqBuf(7, 8)
+      println("length = " + buf.length)
+      println("content = " + buf.element)
+    }
+ 
+Povratni tip metode `newIntSeqBuf` odnosi se na specijalizaciju trejta `Buffer` u kom je tip `U` sada jednak `Int`u.
+Imamo sličan alijas tip u anonimnoj instanci klase u tijelu metode `newIntSeqBuf`.
```
Comment (vjovanov) [inline ba/tutorials/tour/abstract-types.md]: `alijas` is not a word. Maybe use `pseudonim`

File: ba/tutorials/tour/abstract-types.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+      def newIntSeqBuf(e1: Int, e2: Int): SeqBuffer[Int, Seq[Int]] =
+        new SeqBuffer[Int, List[Int]] {
+          val element = List(e1, e2)
+        }
+      val buf = newIntSeqBuf(7, 8)
+      println("length = " + buf.length)
+      println("content = " + buf.element)
+    }
+ 
+Primijetite da moramo koristiti [anotacije za varijancu](variances.html) ovdje;
```
Comment (vjovanov) [inline ba/tutorials/tour/abstract-types.md]: varijancu ~> varijansu

File: ba/tutorials/tour/annotations.md [hunk 11]
```diff
@@ -0,0 +1,132 @@
...
+                while ((c = in.read()) != -1) {
+                    System.out.print((char) c);
+                }
+            } catch (java.io.IOException e) {
+                System.out.println(e.getMessage());
+            }
+        }
+    }
+
+Kada bi zakomentarisali `throws` anotaciju u klasi Reader dobili bi sljedeću grešku s porukom pri kompajliranju Java main programa:
```
Comment (vjovanov) [inline ba/tutorials/tour/annotations.md]: `Reader`

File: ba/tutorials/tour/annotations.md [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+      public String mail();
+    }
+
+I upotrijebiti je kao npr:
+
+    @Source(URL = "http://coders.com/",
+            mail = "support@coders.com")
+    public class MyClass extends HisClass ...
+
+Primjena anotacije u Scali izgleda kao poziv konstruktora, dok se za instanciranje Javinih anotacija moraju koristiti imenovani argumenti:
```
Comment (vjovanov) [inline ba/tutorials/tour/annotations.md]: Scali ~> Skali or Scala-i

File: ba/tutorials/tour/annotations.md [hunk 13]
```diff
@@ -0,0 +1,132 @@
...
+
+    @Source(URL = "http://coders.com/",
+            mail = "support@coders.com")
+    class MyScalaClass ...
+
+Ova sintaksa je ponekad naporna, npr. ako anotacija ima samo jedan element(bez defaultne vrijednosti), pa po konvenciji,
```
Comment (vjovanov) [inline ba/tutorials/tour/annotations.md]: defaultne ~> podrazumevane, predeterminisane

File: ba/tutorials/tour/annotations.md [hunk 14]
```diff
@@ -0,0 +1,132 @@
...
+
+Ova sintaksa je ponekad naporna, npr. ako anotacija ima samo jedan element(bez defaultne vrijednosti), pa po konvenciji, 
+ako se koristi naziv `value` onda se u Javi može koristiti i konstruktor-sintaksa:
+
+    @interface SourceURL {
+        public String value();
+        public String mail() default "";
+    }
+
+I upotrijebiti je kao npr:
```
Comment (vjovanov) [inline ba/tutorials/tour/annotations.md]: npr ~> na primer, npr.

File: ba/tutorials/tour/annotations.md [hunk 15]
```diff
@@ -0,0 +1,132 @@
...
+    @SourceURL("http://coders.com/")
+    public class MyClass extends HisClass ...
+
+U ovom slučaju, Scala omogućuje istu sintaksu:
+
+    @SourceURL("http://coders.com/")
+    class MyScalaClass ...
+
+Element `mail` je specificiran s defaultnom vrijednošću tako da ne moramo eksplicitno navoditi vrijednost za njega. 
+Međutim, ako trebamo, ne možemo miksati dva Javina stila:
```
Comment (vjovanov) [inline ba/tutorials/tour/annotations.md]: miksati ~> mijesati ?

File: ba/tutorials/tour/anonymous-function-syntax.md [hunk 16]
```diff
@@ -0,0 +1,41 @@
...
+
+disqus: true
+
+tutorial: scala-tour
+num: 6
+outof: 33
+language: ba
+---
+
+Scala omogućuje relativno lahku sintaksu za definisanje anonimnih funkcija. Sljedeći izraz kreira funkciju za sljedbenike cijelih brojeva:
```
Comment (vjovanov) [inline ba/tutorials/tour/anonymous-function-syntax.md]: lahku ~> laku

File: ba/tutorials/tour/anonymous-function-syntax.md [hunk 17]
```diff
@@ -0,0 +1,41 @@
...
+
+    (x: Int) => x + 1
+
+Ovo je kratica za definiciju  sljedeće anonimne klase:
```
Comment (vjovanov) [inline ba/tutorials/tour/anonymous-function-syntax.md]: double space

File: ba/tutorials/tour/classes.md [hunk 18]
```diff
@@ -0,0 +1,57 @@
...
+    object Classes {
+      def main(args: Array[String]) {
+        val pt = new Point(1, 2)
+        println(pt)
+        pt.move(10, 10)
+        println(pt)
+      }
+    }
+
+Program definiše izvršnu aplikaciju Classes u form vrhovnog singlton objekta s `main` metodom.
```
Comment (vjovanov) [inline ba/tutorials/tour/classes.md]: `Classes`

File: ba/tutorials/tour/default-parameter-values.md [hunk 19]
```diff
@@ -0,0 +1,72 @@
...
+      /** Kreiraj novu HashMap s defaultnim loadFactor (0.75) */
+      public HashMap(int initialCapacity);
+      public HashMap(int initialCapacity, float loadFactor);
+    }
+
+Ovdje postoje samo dva konstruktora ustvari; jedan koji prima drugu mapu, i jedan koji prima kapacitet i faktor opterećenja.
+Treći i četvrti konstruktor su tu samo da dozvole korisnicima <code>HashMap</code>-e da kreiraju instance s vjerovatno-dobrim default parametrima
+za kapacitet i faktor opterećenja.
+
+Problematičnije je to da su defaultne vrijednosti i u Javadocu *i* u kodu.
```
Comment (vjovanov) [inline ba/tutorials/tour/default-parameter-values.md]: Javadoc-u

File: ba/tutorials/tour/default-parameter-values.md [hunk 20]
```diff
@@ -0,0 +1,72 @@
...
+        */
+      public HashMap();
+      /** Create a new HashMap with default loadFactor (0.75) */
+      public HashMap(int initialCapacity);
+      public HashMap(int initialCapacity, float loadFactor);
+    }
+
+Ovo umanjuje ponavljanje koda, ali je manje ekspresivno.
+
+Scala ima direktnu podršku za ovo:
```
Comment (vjovanov) [inline ba/tutorials/tour/default-parameter-values.md]: What is `ovo` in this case?

File: ba/tutorials/tour/default-parameter-values.md [hunk 21]
```diff
@@ -0,0 +1,72 @@
...
+    val m2= new HashMap[String,Int](20)
+
+    // overriding both
+    val m3 = new HashMap[String,Int](20, 0.8f)
+
+    // override only the loadFactor via
+    // named arguments
+    val m4 = new HashMap[String,Int](loadFactor = 0.8f)
+
+Ovako možemo iskoristiti prednost *bilo koje* defaultne vrijednosti korištenjem [imenovanih parametara]({{ site.baseurl }}/tutorials/tour/named-parameters.html).
```
Comment (vjovanov) [inline ba/tutorials/tour/default-parameter-values.md]: korištenjem ~> korišćenjem ?

File: ba/tutorials/tour/extractor-objects.md [hunk 22]
```diff
@@ -0,0 +1,53 @@
...
+
+disqus: true
+
+tutorial: scala-tour
+num: 15
+outof: 33
+language: ba
+---
+
+U Scali, uzorci(paterni) mogu biti definisani nezavisno od case klasa.
```
Comment (vjovanov) [inline ba/tutorials/tour/extractor-objects.md]: Use english in parenthesis to explain what you refer to.

File: ba/tutorials/tour/implicit-conversions.md [hunk 23]
```diff
@@ -0,0 +1,47 @@
...
+    implicit def list2ordered[A](x: List[A])
+        (implicit elem2ordered: A => Ordered[A]): Ordered[List[A]] =
+      new Ordered[List[A]] { /* .. */ }
+    
+    implicit def int2ordered(x: Int): Ordered[Int] = 
+      new Ordered[Int] { /* .. */ }
+
+Za implicitno importovane objekte `scala.Predef` deklariše nekoliko predefinisanih tipova (npr. `Pair`) i metoda (npr. `assert`) ali i nekoliko implicitnih konverzija.
+
+Naprimjer, kada se pozivaju Javine metode koje očekuju `java.lang.Integer`, možete proslijediti `scala.Int`.
```
Comment (vjovanov) [inline ba/tutorials/tour/implicit-conversions.md]: Na primjer

File: ba/tutorials/tour/lower-type-bounds.md [hunk 24]
```diff
@@ -0,0 +1,58 @@
...
+
+    case class ListNode[+T](h: T, t: ListNode[T]) {
+      def head: T = h
+      def tail: ListNode[T] = t
+      def prepend[U >: T](elem: U): ListNode[U] =
+        ListNode(elem, this)
+    }
+
+_Napomena:_ nova `prepend` metoda ima manje restriktivan tip.
+Ona dozvoljava, naprimjer, da se nadoveže objekt nadtipa na postojeću listu.
```
Comment (vjovanov) [inline ba/tutorials/tour/lower-type-bounds.md]: na primjer

Comment (vjovanov) [inline ba/tutorials/tour/lower-type-bounds.md]: Search and replace this, I will not comment any more.

File: ba/tutorials/tour/mixin-class-composition.md [hunk 25]
```diff
@@ -0,0 +1,55 @@
...
+      private var i = 0
+      def hasNext = i < s.length()
+      def next = { val ch = s charAt i; i += 1; ch }
+    }
+ 
+Željeli bismo iskombinirati funkcionalnosti `StringIterator`-a i `RichIterator`-a u jednoj klasi.
+S jednostrukim nasljeđivanjem i interfejsima ovo je nemoguće, jer obje klase sadrže implementacije članova.
+Scala nam pomaže s _kompozicijom mixin-klasa_.
+Ona dozvoljava programerima da ponovo iskoriste razliku definicija klasa, tj., sve nove definicije koje nisu naslijeđene.
+Ovaj mehanizam omogućuje kombiniranje `StringIterator`-a s `RichIterator`-om, kao u sljedećem testnom programu koji ispisuje kolonu svih karaktera datog stringa.
```
Comment (vjovanov) [inline ba/tutorials/tour/mixin-class-composition.md]: testnom ~> test

File: ba/tutorials/tour/pattern-matching.md [hunk 26]
```diff
@@ -0,0 +1,46 @@
...
+        case y: Int => "scala.Int"
+      }
+      println(matchTest("two"))
+    }
+
+Prvi `case` se podudara ako je `x` cijeli broj `1`.
+Drugi `case` se podudara ako je `x` jednak stringu `"two"`.
+Treći slučaj se sastoji od tipskog uzorka; podudara se sa bilo kojim integerom i povezuje vrijednost selektora `x` s varijablom `y` tipa integer.
+
+Scalin pattern matching je najkorisniji za podudaranje algebarskih tipova koji su izraženi kroz [case klase](case-classes.html).
```
Comment (vjovanov) [inline ba/tutorials/tour/pattern-matching.md]: Skalin or Scala-in. Search and replace in the rest.

File: ba/tutorials/tour/unified-types.md [hunk 27]
```diff
@@ -0,0 +1,57 @@
...
+
+disqus: true
+
+tutorial: scala-tour
+num: 2
+outof: 33
+language: ba
+---
+
+Suprotno od Jave, sve vrijednosti u Scali su objekti (uključujući brojeve i funkcije).
```
Comment (vjovanov) [inline ba/tutorials/tour/unified-types.md]: Suprotno od ~> Nasuprot

File: ba/tutorials/tour/variances.md [hunk 28]
```diff
@@ -0,0 +1,57 @@
...
+
+disqus: true
+
+tutorial: scala-tour
+num: 18
+outof: 33
+language: ba
+---
+
+Scala podržava anotacije varijance tipskih parametara [generičkih klasa](generic-classes.html).
```
Comment (vjovanov) [inline ba/tutorials/tour/variances.md]: varijansu. Search and replace in the text.

**Comment by vjovanov** [comment]
Done; excellent work overall. I would use this translation for Croatian and Serbian as well as it is quite close. Given that we have it now there is no point in translating again in those languages.

File: ba/cheatsheets/index.md (hunk 1 above)
Comment (sake92) [inline ba/cheatsheets/index.md]: Fixed.

File: ba/cheatsheets/index.md (hunk 3 above)
Comment (sake92) [inline ba/cheatsheets/index.md]: I have fixed all problems regarding missing space before parens. Thanks!

File: ba/cheatsheets/index.md (hunk 4 above)
Comment (sake92) [inline ba/cheatsheets/index.md]: Fixed.

File: ba/cheatsheets/index.md (hunk 5 above)
Comment (sake92) [inline ba/cheatsheets/index.md]: I think that "destruktivno" would refer to "destroying" and not to "destructuring". We are not destroying anything with pattern matching so I left that as before.

File: ba/tutorials/tour/abstract-types.md (hunk 8 above)
Comment (sake92) [inline ba/tutorials/tour/abstract-types.md]: I've replaced it with "redefinisati". It seemed more clear to me but if you have some other suggestion I'll change it.

File: ba/tutorials/tour/abstract-types.md (hunk 9 above)
Comment (sake92) [inline ba/tutorials/tour/abstract-types.md]: Fixed.

File: ba/tutorials/tour/abstract-types.md (hunk 10 above)
Comment (sake92) [inline ba/tutorials/tour/abstract-types.md]: Fixed.

File: ba/tutorials/tour/annotations.md (hunk 11 above)
Comment (sake92) [inline ba/tutorials/tour/annotations.md]: Fixed.

File: ba/tutorials/tour/annotations.md (hunk 12 above)
Comment (sake92) [inline ba/tutorials/tour/annotations.md]: I left this one as before, but if you insist I'll change it. The thing is that there are some other words that should be treated same, like "Java" for example. I'd then have to change it also, like "Java-i", right?

File: ba/tutorials/tour/annotations.md (hunk 13 above)
Comment (sake92) [inline ba/tutorials/tour/annotations.md]: Fixed, thanks!

File: ba/tutorials/tour/annotations.md (hunk 14 above)
Comment (sake92) [inline ba/tutorials/tour/annotations.md]: Changed to "npr.:", btw, in Bosnian it is "naprimjer".

File: ba/tutorials/tour/annotations.md (hunk 15 above)
Comment (sake92) [inline ba/tutorials/tour/annotations.md]: Fixed. The word "mixin" was stuck in my head... xD  Also, "mixable" is awkward to translate, "umjesiv"?

File: ba/tutorials/tour/anonymous-function-syntax.md (hunk 16 above)
Comment (sake92) [inline ba/tutorials/tour/anonymous-function-syntax.md]: Nope, Bosnian is with "h".

File: ba/tutorials/tour/anonymous-function-syntax.md (hunk 17 above)
Comment (sake92) [inline ba/tutorials/tour/anonymous-function-syntax.md]: Fixed.

File: ba/tutorials/tour/classes.md (hunk 18 above)
Comment (sake92) [inline ba/tutorials/tour/classes.md]: Fixed.

File: ba/tutorials/tour/default-parameter-values.md (hunk 19 above)
Comment (sake92) [inline ba/tutorials/tour/default-parameter-values.md]: Fixed.

File: ba/tutorials/tour/default-parameter-values.md (hunk 21 above)
Comment (sake92) [inline ba/tutorials/tour/default-parameter-values.md]: No, Bosnian is with "t", like "sretan" etc.

File: ba/tutorials/tour/extractor-objects.md (hunk 22 above)
Comment (sake92) [inline ba/tutorials/tour/extractor-objects.md]: Fixed.

File: ba/tutorials/tour/implicit-conversions.md (hunk 23 above)
Comment (sake92) [inline ba/tutorials/tour/implicit-conversions.md]: "Naprimjer" is in right in Bosnian.

File: ba/tutorials/tour/mixin-class-composition.md (hunk 25 above)
Comment (sake92) [inline ba/tutorials/tour/mixin-class-composition.md]: Fixed.

File: ba/tutorials/tour/unified-types.md (hunk 27 above)
Comment (sake92) [inline ba/tutorials/tour/unified-types.md]: Fixed.

File: ba/tutorials/tour/default-parameter-values.md (hunk 20 above)
Comment (sake92) [inline ba/tutorials/tour/default-parameter-values.md]: I missed this one, fixed it in third commit. Literal translation from English version, also ambiguous.

**Comment by sake92** [comment]
Thanks for reviewing this, I appreciate it. I have fixed all of the formatting issues, reviewed all documents again, used more consistent terminology. If there are other issues regarding this, please let me know.

File: ba/tutorials/tour/abstract-types.md (hunk 8 above)
Comment (vjovanov) [inline ba/tutorials/tour/abstract-types.md]: In german they say `prepisati` (überschreiben) also in some university books this is the term of choice. I would personally use `premostiti`.

File: ba/tutorials/tour/annotations.md (hunk 12 above)
Comment (vjovanov) [inline ba/tutorials/tour/annotations.md]: I think so but not 100% sure. It is fine to stay as before. I like Skali the most as this is how you pronounce it.

File: ba/tutorials/tour/annotations.md (hunk 15 above)
Comment (vjovanov) [inline ba/tutorials/tour/annotations.md]: :) This one is hard. I would not use mixable in the first place; can be mixed together is fine. I 6 years with Scala I have never heard that word.

**Comment by vjovanov** [comment]
Excellent work!

**Comment by SethTisue** [comment]
thanks all!

> I would use this translation for Croatian and Serbian as well as it is quite close

(hmm, I'm not sure what the best way to do that within the existing site infrastructure would be)

**Pull request merged**